tests/PHPUnit/Unit/ConfigTest.php

```diff
@@ -347,19 +268,23 @@ public function getDumpConfigData()
                 array('General' => array('debug' => 0),       // global
                       'Tracker' => array('anonymize' => 0)),
                 array(),                                      // common
-                array('Tracker' => array('anonymize' => 2)),
+                array('Tracker' => array('anonymize' => 2),
+                      'General' => array('debug' => 1)),
                 $header . "[General]\ndebug = 1\n\n[Tracker]\nanonymize = 2\n\n",
             )),
 
-            array('sort, common sections before new section', array(
+            /*TODO: is this test needed?
```

Comment:
It tests that the order of the sections is first sections that have data in common w/ global + config, and then data that is only in config.ini.php. I don't think I can easily replicate this behavior, and I'm not sure if it's important. Is it?